src/site/content/en/blog/url/index.md

```diff
@@ -0,0 +1,296 @@
+---  
+title: What are the parts of a URL?  
+subhead: "What's the difference between a domain name, hostname, site and origin? This article explains."
+authors:  
+  - samdutton  
+date: 2023-02-22
+hero: image/80mq7dk16vVEg8BBhsVe42n6zn82/EAQ6QLeBBonbpyqeGWPb.jpg
+alt:   
+tags:  
+    - blog
+    - html
+    - network
+    - privacy
+---
+
+Most of the time it's fine to be relaxed about naming websites and URLs. It's
+usually OK to say things like "I bought a domain name" or "Our images are hosted on a different
+site," even if that's not strictly true.
+
+However, in some contexts it's crucial to be accurate and specific when referring to the parts of a
+URL. For example, when dealing with cookies, you need to understand the difference between
+[site](#site) and [origin](#origin). This article is designed to help with that.
+
+The explanations here are only intended as a guide:
+[The URL Living Standard](https://url.spec.whatwg.org/) provides formal definitions of URL parts.
+
+{% Aside %}
+The named parts of a URL may coincide, but that doesn't mean they're equivalent!   
+
+The [FQDN](#fqdn), [eTLD+1](#etld1), [hostname](#hostname) and [registrable domain](#registrable-domain)
```

Comment:
Likewise, I think it's best for us to merge "eTLD+1" and "registrable domain" into the latter, following URL. Again, it might be reasonable to note that lots of people will call it "eTLD+1", but we should encourage alignment on one term.

Reply:
Thanks Mike (belatedly!) I've actually removed this bit — I think the aside was more confusing than useful anyway.